wiki/backup.md

```diff
@@ -0,0 +1,85 @@
+# Backing up data
+
+This page aims to explain how to back up your data to either your cloud or an external drive, why you should regularly back up your files.
```

Comment:
```suggestion
This page aims to explain how to back up your data to either your cloud or an external drive, and why you should regularly back up your files.
```

Reply:
I was thinking of adding this change for a bit but I was like "nah let's wait for the review"